package.json

```diff
@@ -40,7 +40,7 @@
     "lint:gherkin": "gherkin-lint",
     "pretest:integration": "preview",
     "test:integration": "run-s 'test:integration:base -- --profile noWip'",
-    "test:integration:base": "NODE_OPTIONS=--enable-source-maps DEBUG=any cucumber-js test/integration --profile base",
+    "test:integration:base": "DEBUG=any cucumber-js test/integration --profile base",
```

Comment:
had to remove the `NODE_OPTIONS=--enable-source-maps` in order to verify against v10 since the flag isnt allowed there